src/main.ts

```diff
@@ -105,7 +107,19 @@ async function runCargoSemverChecks(cargo: rustCore.Cargo): Promise<void> {
     // need to set the target directory explicitly.
     process.env["CARGO_TARGET_DIR"] = CARGO_TARGET_DIR;
 
-    await cargo.call(["semver-checks", "check-release"].concat(getCheckReleaseArguments()));
+    const options: any = {};
+    options.listeners = {
+        stdout: (data: Buffer) => {
+            SEMVER_CHECKS_OUTPUT += data.toString();
+        },
+        stderr: (data: Buffer) => {
+            SEMVER_CHECKS_OUTPUT += data.toString();
+        },
```

Comment:
Consider checking if `data` is non-empty and not undefined here. The `undefined` prefix we see in the output right now is probably from an invocation with `data = undefined` here.

Reply:
5fd6fc61d96d24b861fd7cf568b1bd4cb6aa1b9a